core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala

```diff
@@ -28,6 +28,8 @@ package org.apache.spark.internal.config
  * value declared as a string.
  *
  * @param key the key for the configuration
+ * @param prependedKey the key for the configuration which prepended
+ * @param prependSeparator the separator which is used for prepend
```

Comment:
which will be prepended
which is used for prepending

Reply:
Fixed.